DTS-STN/dts-rmp
# Engagement and Contact ListItem Responsiveness

# Description

[[Bug] Mobile view - Search engagement/contact page](https://taiga.dts-stn.com/project/pond-knowlege-management-portal/issue/245?kanban-status=100)

This PR enhances the mobile experience for Contact and Engagement ListItems. Also deleted unused .vue files.

## Test Instructions

1. Navigate to Contact and Engagement search pages
1. Use dev tools to change the size of the display down to 375 px width (iPhone 6/7/8 size and likely the smallest to expect)
1. See that elements of the ListItems are not cut off by the display

## Help Requested

- Any additional fixes you think I should add
